plat/allwinner/common/sunxi_cpu_ops.c

```diff
@@ -11,8 +11,6 @@
 #include <sunxi_cpucfg.h>
 #include <utils_def.h>
 
-#include "sunxi_private.h"
```

Comment:
This file is including `sunxi_private.h` because it defines functions declared there.

Reply:
Yes, I always scratch my head about this header file. `sunxi_private.h` hints at that it contains functions private to the Allwinner platform, but its placement in the common/ directory makes it actually private to the _common_ code only. Thus the introduction of `sunxi_helpers.h` in a proper include directory to fix this. Should we move all prototypes into this one, renaming it to `sunxi_private.h`, then just include only this file from everywhere? The distinction between the two seems rather arbitrary to me.